policyengine_us/variables/gov/local/co/denver/dhs/has_co_denver_dhs_elderly_disabled.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+
+
+class has_co_denver_dhs_elderly_disabled(Variable):
+    value_type = bool
+    entity = SPMUnit
+    definition_period = YEAR
+    documentation = "Whether the SPM unit has a person who meets Denver DHS definitions of elderly or disabled"
+    label = "Has Denver DHS elderly or disabled people"
+
+    def formula(spm_unit, period, parameters):
+        person = spm_unit.members
+        elderly = person("is_co_denver_dhs_elderly", period)
```

Comment:
Can I use `is_senior` to replace `is_co_denver_dhs_elderly`

Reply:
If the program documents include the number 65, we need to specify it as a policy parameter (as in all numbers that we leverage that are part of the law)